### PR TITLE
KCRO-190: Additional APIs to consume pvt_chat.getstream in user doc

### PR DESCRIPTION
**Pull request by Navoda626**
Adding tests for the following APIs

**GET accounts/{ACCOUNT_ID}/users/getstream** <-- list all user documents with pvt_chat.getstream.enabled == true

**GET accounts/{ACCOUNT_ID}/users/{USER_ID}/getstream** <-- get the getstream object of a specific user

**PUT accounts/{ACCOUNT_ID}/users/{USER_ID}/getstream** <-- sets pvt_chat.getstream.enabled = true and set any object in json data into getstream object for the USER_ID and return the pvt_chat.getsream object

**PATCH accounts/{ACCOUNT_ID}/users/{USER_ID}/getstream** <-- merge the json data with pvt_chat.getstream (overwriting existing parameters including the value of enabled ) and return the pvt_chat.getsream object

**DELETE accounts/{ACCOUNT_ID}/users/{USER_ID}/getstream** <-- removes the property pvt_chat.getstream.enabled for the USER_ID and return the deleted pvt_chat.getsream object

### CODE REVIEW

**Comment by nkarunarathne** [comment]
Looks good

File: src/seq/seq_getstream.erl [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+-define(ACCOUNT_NAME, list_to_binary([?MODULE_STRING "_", kz_term:to_binary(?FUNCTION_NAME)])).
+
+-spec seq() -> 'ok'.
+seq() ->
+    Fs = [fun seq_getstream/0],
+    run_funcs(Fs).
+
+run_funcs([]) -> 'ok';
+run_funcs([F|Fs]) ->
+    cleanup(),
```
Comment (jamesaimonetti) [inline src/seq/seq_getstream.erl]: Each seq_* should cleanup after itself after running successfully. No need for double cleanup here

File: src/seq/seq_getstream.erl [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    _ = F(),
+    cleanup(),
+    run_funcs(Fs).
+
+-spec seq_getstream() -> 'ok'.
+seq_getstream() ->
+    API = pqc_cb_api:init_api(['crossbar'], ['cb_websites']),
```
Comment (jamesaimonetti) [inline src/seq/seq_getstream.erl]: Have you actually run this test? You start cb_websites here but API calls lower down go to `cb_getstream`

File: src/seq/seq_getstream.erl (hunk 1 above)
Comment (Navoda626) [reply src/seq/seq_getstream.erl]: Got it. Thanks. Will remove the initial cleanup()

File: src/seq/seq_getstream.erl (hunk 2 above)
Comment (Navoda626) [reply src/seq/seq_getstream.erl]: Yes test ran successfully locally, but now wonder how that happened.
Will change this to cb_getstream

File: src/seq/seq_getstream.erl [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+-spec seq() -> 'ok'.
+seq() ->
+    Fs = [fun seq_getstream/0],
+    run_funcs(Fs).
+
+run_funcs([]) -> 'ok';
+run_funcs([F|Fs]) ->
+    _ = F(),
+    cleanup(),
```
Comment (jamesaimonetti) [inline src/seq/seq_getstream.erl]: Extra `cleanup()` isn't necessary; if `F` runs successfully, all is cleaned up before the rest of `Fs` are run. If F crashes, no cleanup needed as we need the system unchanged to inspect where the failure occurred.
`cleanup/0` is called after a test failure to reset everything before running `seq` tests again.

**Review by jamesaimonetti — CHANGES_REQUESTED** [review]

File: src/seq/seq_getstream.erl (hunk 3 above)
Comment (Navoda626) [reply src/seq/seq_getstream.erl]: Got it! Thank you James! I will remove the extra `cleanup/0` function

File: src/seq/seq_getstream.erl (hunk 1 above)
Comment (sanjeevie6353) [reply src/seq/seq_getstream.erl]: This has been fixed.

File: src/seq/seq_getstream.erl (hunk 3 above)
Comment (sanjeevie6353) [reply src/seq/seq_getstream.erl]: This has been fixed.

**Pull request merged**